packages/react-ui-kit/demo/Demo.jsx

```diff
@@ -215,10 +215,10 @@ export default function Demo() {
               <Input placeholder="Placeholder" type="email" required />
               <Button type="submit">Submit Button</Button>
               <InputBlock>
-                <Input placeholder="InputBlock" />
+                <Input placeholder="InputBlock" markInvalid />
```

Comment:
what iz daz?

Reply:
a helper prop, in some cases we (and the design guys) want a little indicator if there is an error in this input field. the native :invalid selector unfortunately doesn't cut it.